L1Trigger/L1TMuonEndCap/interface/EndCapParamsHelper.h

```diff
@@ -39,7 +39,7 @@ namespace l1t {
   class EndCapParamsHelper {
```

Comment:
No changes to this file except spaces ... will probably make the review easier if we just leave it untouched.